dell/e7240/default.nix

```diff
@@ -1,6 +0,0 @@
-{
-  imports = [
```

Comment:
Can we add a deprecation warning here and import the new module here?
This is a bit nicer for existing users.

Check: `common/pc/hdd/default.nix` for an example for deprecation.